server.js

```diff
@@ -41,19 +41,31 @@ const checkCredentials = async () => {
         throw new Error(error);
     }
 };
+
+//Important!
```

Comment:
It would be great to put a little attention to how we format code comments:

```ts
//Some comment

vs 

// Some comment with space before the first letter

//Multi line
//comment without a space at the beginning
// and starting with slashes 

vs

/*
  Multiline (block) comments with proper spacing
  make it easier to read.  
*/

```